### ISSUE TITLE
General error when trying to back up after fresh install.

### ISSUE THREAD

**Issue by Park8706** [grokability/snipe-it#13179]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

When I attempt to generate a back up it gives me a failure error as I will show below.

### Reproduction steps

1.Create back up
2.failure
3.
...

### Expected behavior

Back up created

### Snipe-IT Version

6.0.14

### Operating System

Server 2019

### Web Server

IIS

### PHP Version

7.4.33

### Operating System

Windows

### Error messages

```shell
Error Backup failed because The dump process failed with exitcode 1 : General error : . #0 C:\inetpub\wwwroot\snipe-it\vendor\spatie\db-dumper\src\DbDumper.php(263): Spatie\DbDumper\Exceptions\DumpFailed::processDidNotEndSuccessfully() #1 C:\inetpub\wwwroot\snipe-it\vendor\spatie\db-dumper\src\Databases\MySql.php(203): Spatie\DbDumper\DbDumper->checkIfDumpWasSuccessFul() #2 C:\inetpub\wwwroot\snipe-it\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(260): Spatie\DbDumper\Databases\MySql->dumpToFile() #3 [internal function]: Spatie\Backup\Tasks\Backup\BackupJob->Spatie\Backup\Tasks\Backup\{closure}() #4 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Collections\Collection.php(695): array_map() #5 C:\inetpub\wwwroot\snipe-it\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(263): Illuminate\Support\Collection->map() #6 C:\inetpub\wwwroot\snipe-it\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(171): Spatie\Backup\Tasks\Backup\BackupJob->dumpDatabases() #7 C:\inetpub\wwwroot\snipe-it\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(147): Spatie\Backup\Tasks\Backup\BackupJob->createBackupManifest() #8 C:\inetpub\wwwroot\snipe-it\vendor\spatie\laravel-backup\src\Commands\BackupCommand.php(56): Spatie\Backup\Tasks\Backup\BackupJob->run() #9 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(36): Spatie\Backup\Commands\BackupCommand->handle() #10 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Container\Util.php(40): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}() #11 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(93): Illuminate\Container\Util::unwrapIfClosure() #12 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(37): Illuminate\Container\BoundMethod::callBoundMethod() #13 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Container\Container.php(653): Illuminate\Container\BoundMethod::call() #14 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Console\Command.php(136): Illuminate\Container\Container->call() #15 C:\inetpub\wwwroot\snipe-it\vendor\symfony\console\Command\Command.php(298): Illuminate\Console\Command->execute() #16 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Console\Command.php(121): Symfony\Component\Console\Command\Command->run() #17 C:\inetpub\wwwroot\snipe-it\vendor\spatie\laravel-backup\src\Commands\BaseCommand.php(16): Illuminate\Console\Command->run() #18 C:\inetpub\wwwroot\snipe-it\vendor\symfony\console\Application.php(1024): Spatie\Backup\Commands\BaseCommand->run() #19 C:\inetpub\wwwroot\snipe-it\vendor\symfony\console\Application.php(299): Symfony\Component\Console\Application->doRunCommand() #20 C:\inetpub\wwwroot\snipe-it\vendor\symfony\console\Application.php(171): Symfony\Component\Console\Application->doRun() #21 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Console\Application.php(94): Symfony\Component\Console\Application->run() #22 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Console\Application.php(186): Illuminate\Console\Application->run() #23 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(263): Illuminate\Console\Application->call() #24 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(261): Illuminate\Foundation\Console\Kernel->call() #25 C:\inetpub\wwwroot\snipe-it\app\Http\Controllers\SettingsController.php(1110): Illuminate\Support\Facades\Facade::__callStatic() #26 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(54): App\Http\Controllers\SettingsController->postBackups() #27 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(45): Illuminate\Routing\Controller->callAction() #28 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Routing\Route.php(262): Illuminate\Routing\ControllerDispatcher->dispatch() #29 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Routing\Route.php(205): Illuminate\Routing\Route->runController() #30 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Routing\Router.php(721): Illuminate\Routing\Route->run() #31 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(128): Illuminate\Routing\Router->Illuminate\Routing\{closure}() #32 C:\inetpub\wwwroot\snipe-it\app\Http\Middleware\CheckPermissions.php(24): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #33 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): App\Http\Middleware\CheckPermissions->handle() #34 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Session\Middleware\AuthenticateSession.php(58): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #35 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Session\Middleware\AuthenticateSession->handle() #36 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Auth\Middleware\Authenticate.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #37 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Auth\Middleware\Authenticate->handle() #38 C:\inetpub\wwwroot\snipe-it\app\Http\Middleware\AssetCountForSidebar.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #39 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): App\Http\Middleware\AssetCountForSidebar->handle() #40 C:\inetpub\wwwroot\snipe-it\vendor\laravel\passport\src\Http\Middleware\CreateFreshApiToken.php(50): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #41 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Laravel\Passport\Http\Middleware\CreateFreshApiToken->handle() #42 C:\inetpub\wwwroot\snipe-it\app\Http\Middleware\CheckForTwoFactor.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #43 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): App\Http\Middleware\CheckForTwoFactor->handle() #44 C:\inetpub\wwwroot\snipe-it\app\Http\Middleware\CheckUserIsActivated.php(47): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #45 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): App\Http\Middleware\CheckUserIsActivated->handle() #46 C:\inetpub\wwwroot\snipe-it\app\Http\Middleware\CheckLocale.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #47 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): App\Http\Middleware\CheckLocale->handle() #48 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(78): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #49 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle() #50 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #51 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle() #52 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(67): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #53 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Cookie\Middleware\EncryptCookies->handle() #54 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #55 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Routing\Router.php(723): Illuminate\Pipeline\Pipeline->then() #56 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Routing\Router.php(698): Illuminate\Routing\Router->runRouteWithinStack() #57 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Routing\Router.php(662): Illuminate\Routing\Router->runRoute() #58 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Routing\Router.php(651): Illuminate\Routing\Router->dispatchToRoute() #59 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(167): Illuminate\Routing\Router->dispatch() #60 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(128): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}() #61 C:\inetpub\wwwroot\snipe-it\vendor\livewire\livewire\src\DisableBrowserCache.php(19): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #62 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Livewire\DisableBrowserCache->handle() #63 C:\inetpub\wwwroot\snipe-it\vendor\barryvdh\laravel-debugbar\src\Middleware\InjectDebugbar.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #64 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Barryvdh\Debugbar\Middleware\InjectDebugbar->handle() #65 C:\inetpub\wwwroot\snipe-it\vendor\fruitcake\laravel-cors\src\HandleCors.php(38): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #66 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Fruitcake\Cors\HandleCors->handle() #67 C:\inetpub\wwwroot\snipe-it\app\Http\Middleware\PreventBackHistory.php(23): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #68 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): App\Http\Middleware\PreventBackHistory->handle() #69 C:\inetpub\wwwroot\snipe-it\app\Http\Middleware\SecurityHeaders.php(26): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #70 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): App\Http\Middleware\SecurityHeaders->handle() #71 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\TransformsRequest.php(21): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #72 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull.php(31): Illuminate\Foundation\Http\Middleware\TransformsRequest->handle() #73 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull->handle() #74 C:\inetpub\wwwroot\snipe-it\app\Http\Middleware\CheckForDebug.php(25): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #75 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): App\Http\Middleware\CheckForDebug->handle() #76 C:\inetpub\wwwroot\snipe-it\app\Http\Middleware\CheckForSetup.php(25): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #77 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): App\Http\Middleware\CheckForSetup->handle() #78 C:\inetpub\wwwroot\snipe-it\vendor\fideloper\proxy\src\TrustProxies.php(57): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #79 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Fideloper\Proxy\TrustProxies->handle() #80 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #81 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\View\Middleware\ShareErrorsFromSession->handle() #82 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(121): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #83 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(64): Illuminate\Session\Middleware\StartSession->handleStatefulRequest() #84 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Session\Middleware\StartSession->handle() #85 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\PreventRequestsDuringMaintenance.php(86): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #86 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): Illuminate\Foundation\Http\Middleware\PreventRequestsDuringMaintenance->handle() #87 C:\inetpub\wwwroot\snipe-it\app\Http\Middleware\NoSessionStore.php(28): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #88 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(167): App\Http\Middleware\NoSessionStore->handle() #89 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}() #90 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(142): Illuminate\Pipeline\Pipeline->then() #91 C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(111): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter() #92 C:\inetpub\wwwroot\snipe-it\public\index.php(52): Illuminate\Foundation\Http\Kernel->handle() #93 {main} Backup failed because: The dump process failed with exitcode 1 : General error : .
```

### Additional context

I had been running  Snipe-IT fine until yesterday when they uninstalled windows update server from the same server and it killed my database somehow. After trying to get snipe-it back up I finally gave up and removed IIS and uninstalled all software as laid out in the guide for installing onto an IIS to start over. After getting everything back up with the same versions ect as before I am getting an error when I attempt to make a backup. Unlike last time where it said invalid path it just gives me it failed. I am unsure as to what the issue is this time.

Edit: Also when I try and restore from a backup I made before this all happened it kicks me back to the start of the pre-flight steps.
![Screenshot 2023-06-16 161326](https://github.com/snipe/snipe-it/assets/134626149/b4c70e6f-bf66-4ef6-9120-2c66140401b2)

**Comment by devvonmonahan** [comment]
Can you try a manual run of mysqldump by opening up command prompt & changing directory to where mysqldump is located.
I think the command would be; mysqldump -u [username] -p[password] snipeit > /backup/path/backup
There is no space between -p and the password.

**Comment by Park8706** [comment]
> Can you try a manual run of mysqldump by opening up command prompt & changing directory to where mysqldump is located. I think the command would be; mysqldump -u [username] -p[password] snipeit > /backup/path/backup There is no space between -p and the password.

For the path do I put what I have in for the DB Dump path in my env file?.

Update: while waiting for response on this I did run the php command line to make a back up and it created a backup file that way. So my issue now is not being able to make one via the GUI and to restore from my old backup to restore my database.

Here is my cmd line of me creating a backup then it failing to restore from my old backup which holds all the old database stuff I need to restore.

![isssue2](https://github.com/snipe/snipe-it/assets/134626149/8c9349ca-9cd7-4031-ae15-b073a13ef96f)

**Comment by Park8706** [comment]
> > Can you try a manual run of mysqldump by opening up command prompt & changing directory to where mysqldump is located. I think the command would be; mysqldump -u [username] -p[password] snipeit > /backup/path/backup There is no space between -p and the password.
> 
> For the path do I put what I have in for the DB Dump path in my env file?.
> 
> Update: while waiting for response on this I did run the php command line to make a back up and it created a backup file that way. So my issue now is not being able to make one via the GUI and to restore from my old backup to restore my database.
> 
> Here is my cmd line of me creating a backup then it failing to restore from my old backup which holds all the old database stuff I need to restore.
> 
> ![isssue2](https://user-images.githubusercontent.com/134626149/246845344-8c9349ca-9cd7-4031-ae15-b073a13ef96f.png)

Ok update from all this. I still am unable to backup from GUI but as stated I can from command line. I was able to figure out why I could not restore from my old backup. I had to edit my RestoreFromBackup.php to point to mysql.exe not mysql.
Why this was an issue this time I am unsure but it was a fix I found here from a previous issue. I was able to restore via the command line. Still would like to figure out why I can't backup from the Gui but I am hopeful this will help narrow down the issue for anyone trying to help.

**Comment by josercolon** [comment]
> > > Can you try a manual run of mysqldump by opening up command prompt & changing directory to where mysqldump is located. I think the command would be; mysqldump -u [username] -p[password] snipeit > /backup/path/backup There is no space between -p and the password.
> > 
> > 
> > For the path do I put what I have in for the DB Dump path in my env file?.
> > Update: while waiting for response on this I did run the php command line to make a back up and it created a backup file that way. So my issue now is not being able to make one via the GUI and to restore from my old backup to restore my database.
> > Here is my cmd line of me creating a backup then it failing to restore from my old backup which holds all the old database stuff I need to restore.
> > ![isssue2](https://user-images.githubusercontent.com/134626149/246845344-8c9349ca-9cd7-4031-ae15-b073a13ef96f.png)
> 
> Ok update from all this. I still am unable to backup from GUI but as stated I can from command line. I was able to figure out why I could not restore from my old backup. I had to edit my RestoreFromBackup.php to point to mysql.exe not MySQL. Why this was an issue this time I am unsure but it was a fix I found here from a previous issue. I was able to restore via the command line. Still would like to figure out why I can't backup from the GUI but I am hopeful this will help narrow down the issue for anyone trying to help.

Were you able to fix this? Same thing happened to us after 6.1.1 upgrade and fresh install. We are in the process of migrating Server 2012 R2 Servers to  Server 2022 so we decided to patch snipe it to the latest then try to restore the data on the same version our fresh install would be on. But we get a 500 | Server error and the backup does not run. Same thing happens when trying to restore on the fresh install. We are able to backup using php artisan snipeit:backup with no errors and were able to restore using export to SQL using HeidiSQL and then importing it on the 2022 server and have every else working just fine except GUI backups. Its does the same thing on both the upgrade and the fresh install. We are using Maria DB, PHP 8.1.2, ISS and Server 2022 on the fresh install.

**Comment by Park8706** [comment]
> > > > Can you try a manual run of mysqldump by opening up command prompt & changing directory to where mysqldump is located. I think the command would be; mysqldump -u [username] -p[password] snipeit > /backup/path/backup There is no space between -p and the password.
> > > 
> > > 
> > > For the path do I put what I have in for the DB Dump path in my env file?.
> > > Update: while waiting for response on this I did run the php command line to make a back up and it created a backup file that way. So my issue now is not being able to make one via the GUI and to restore from my old backup to restore my database.
> > > Here is my cmd line of me creating a backup then it failing to restore from my old backup which holds all the old database stuff I need to restore.
> > > ![isssue2](https://user-images.githubusercontent.com/134626149/246845344-8c9349ca-9cd7-4031-ae15-b073a13ef96f.png)
> > 
> > 
> > Ok update from all this. I still am unable to backup from GUI but as stated I can from command line. I was able to figure out why I could not restore from my old backup. I had to edit my RestoreFromBackup.php to point to mysql.exe not MySQL. Why this was an issue this time I am unsure but it was a fix I found here from a previous issue. I was able to restore via the command line. Still would like to figure out why I can't backup from the GUI but I am hopeful this will help narrow down the issue for anyone trying to help.
> 
> Were you able to fix this? Same thing happened to us after 6.1.1 upgrade and fresh install. We are in the process of migrating Server 2012 R2 Servers to Server 2022 so we decided to patch snipe it to the latest then try to restore the data on the same version our fresh install would be on. But we get a 500 | Server error and the backup does not run. Same thing happens when trying to restore on the fresh install. We are able to backup using php artisan snipeit:backup with no errors and were able to restore using export to SQL using HeidiSQL and then importing it on the 2022 server and have every else working just fine except GUI backups. Its does the same thing on both the upgrade and the fresh install. We are using Maria DB, PHP 8.1.2, ISS and Server 2022 on the fresh install.

Still can't back up in snipe-it and have to use the command line to do so. As for restoring I was able to do it in cmd via the commands in the documentation but I haven't done it in the web browser as the file is now over 2mb and I have read once you are over that its best do it via command line anyway. If the commands they list in the documentation don't work I would suggest going to the RestoreFromBackup.php file in the app/console/commands folder in snipe-it and editing the line where it starts with $mysql_binary = config. Find the part with '/mysql' and add .exe to the mysql portion. That fix worked for us and seems to be holding.

**Comment by josercolon** [comment]
Our file is about 30mb, I just transferred the file from the old server to the new server instead of using the web UI. When I tried to restore it this way the same thing happened to us, same error and if you refresh the site it took us back to the setup page (pre flight). The SQL file export method was pretty straight forward. So far we have no other issue except GUI backups.

**Comment by josercolon** [comment]
Fyi, GUI backup was fixed in version 6.1.2 for us. Issue #13230 seems was the culprit.
